test/elvis_SUITE.erl

```diff
@@ -119,9 +119,8 @@ main_config(_Config) ->
         OptFun = fun() -> elvis:main("-c") end,
         _ = check_first_line_output(OptFun, Expected, fun matches_regex/2),
 
-        EnoentExpected = "enoent",
         OptEnoentFun = fun() -> elvis:main("-c missing") end,
-        _ = check_first_line_output(OptEnoentFun, EnoentExpected, fun matches_regex/2),
+        ok = OptEnoentFun(),
```

Comment:
This can also be removed. Where `elvis_core` was failing with an exception now it silently eats it up (this was a conscious choice), so the test had to be "adapted".

Reply:
Yeah, remove this one.

Reply:
9c3ee12